README.adoc

```diff
@@ -305,11 +305,11 @@ a modified file in the correct place. Just commit it and push the change.
 If you don't have an IDE preference we would recommend that you use
 http://www.springsource.com/developer/sts[Spring Tools Suite] or
 http://eclipse.org[Eclipse] when working with the code. We use the
-http://eclipse.org/m2e/[m2eclipe] eclipse plugin for maven support. Other IDEs and tools
+http://eclipse.org/m2e/[m2eclipse] eclipse plugin for maven support. Other IDEs and tools
```

Comment:
These fixes for m2eclipse actually have their source here: https://github.com/spring-cloud/spring-cloud-build/blob/master/docs/src/main/asciidoc/building-base.adoc